frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -241,11 +241,45 @@ const SubscriptionStatus = ({ muted = false, subscription }) => {
   );
 };
 
+const getLinkForInstallPage = (
+  subscription: SubscriptionKind,
+  namespace: string,
+  obj: ClusterServiceVersionKind | SubscriptionKind,
+) => {
+  const installPageRoute = `/operatorhub/install?pkg=${_.get(
```

Comment:
Avoid `_.get`. Use optional chaining (here and below).

Should this not be called `subscription` instead of `pkg`?